zstash/hpss_new.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import print_function, absolute_import
+
+import os.path
+import subprocess
+from .settings import logger
+
+
+def run_command(command, error_str):
+    completed_process = subprocess.run(command.split(), capture_output=True, encoding='utf-8')
+    status = completed_process.return_code
+    if status != 0:
+        logger.error(error_str)
+        logger.debug('stdout:\n%s', completed_process.stdout)
+        logger.debug('stderr:\n%s', completed_process.stderr)
+        raise Exception
+
+
+def hpss_transfer(hpss, file_path, put, keep=None):
+    if hpss == 'none':
```

Comment:
Should we make 'none' cas insensitive, just in case someone uses Python capitalization None.